### DISCUSSION TITLE
How to exclude specific line from spotless check

### DISCUSSION THREAD

**Discussion by shadikkhan** [cqfn/diKTat#1607] [Q&A] [answered]
Sometime I have situation where I need var name in uppercase in model (e.g. someURL OR PCI) but spotless throw errors for these and I did not find any solution to exclude specific line(s) from spotless check.

Is there anything available in diktat/spotless?

**Comment by orchestr7** [comment, marked as answer]
Hey, how about `@Suppress` annotation in code?

https://github.com/saveourtool/diktat#-suppress-warningsinspections
